en/sql/query/show.rst

```diff
@@ -1837,3 +1837,67 @@ Num_busy_workers            INT             The number of busy worker threads of
 Num_connection_workers      INT             The number of connection worker threads of the queue
 =========================== =============== =======================================================
 
+SHOW PAGE BUFFER STATUS
+-----------------------
+
+It shows the status of data page buffer pool.
+
+.. code-block:: sql
+
+    SHOW PAGE BUFFER STATUS;
+
+This query has the following columns:
+
+=========================== =============== ================================================================================
+Column name                 Type            Description
+=========================== =============== ================================================================================
+Hit_rate                    NUMERIC(13,10)  The buffer pool hit rate (since the last printout)
+Num_hit                     BIGINT          The number of buffer hit (since the last printout)
+Num_page_request            BIGINT          The number of page requests (since the last printout)
+Pool_size                   INT             Buffer pool size in pages
+Page_size                   INT             Data page size
+Free_pages                  INT             The number of free pages in the buffer pool
+Victim_candidate_pages      INT             The number of victim candidate pages in the cold LRU area
+Clean_pages                 INT             The number of clean pages in the buffer pool
+Dirty_pages                 INT             The number of dirty pages in the buffer pool
+Num_index_pages             INT             The number of index pages in the buffer pool
+Num_data_pages              INT             The number of data pages in the buffer pool
+Num_system_pages            INT             The number of system pages in the buffer pool
+Num_temp_pages              INT             The number of temp pages in the buffer pool
+Num_pages_created           BIGINT          The number of pages created in the buffer pool (since the last printout)
+Num_pages_written           BIGINT          The number of pages written to disk in the buffer pool (since the last printout)
+Pages_written_rate          NUMERIC(20,10)  The number of pages written per second (since the last printout)
+Num_pages_read              BIGINT          The number of pages read from disk in the buffer pool (since the last printout)
+Pages_read_rate             NUMERIC(20,10)  The number of pages read per second (since the last printout)
+Num_flusher_waiting_threads INT             The number of waiting threads for flusher
+=========================== =============== ================================================================================
+
+The following shows the examples of this syntax.
```

Comment:
```suggestion
The following shows examples of this syntax.
```

Reply:
```suggestion
The following shows the example of this syntax.
```
I think this sentence seems to be for the consistency with other example descriptions

Reply:
@charischau As @JoohoK said, this is a copy of another part of the manual. Would you like to fix this? I will follow your opinion.

Reply:
For consistency, I think is better to change it as @JoohoK said.  
```suggestion
The following shows the example of this syntax.
```

Reply:
@charischau In the manual, "examples" are used, not "example". So, would I have to modify it?

Reply:
I see, if 'examples' are used, then 'the' should be deleted. 
```suggestion
The following shows examples of this syntax.
```